internal/cli/deployments/logs_test.go

```diff
@@ -112,3 +112,26 @@ func TestLogs_RunAtlas(t *testing.T) {
 		t.Fatalf("Run() unexpected error: %v", err)
 	}
 }
+
+func TestLogs_PostRun(t *testing.T) {
```

Comment:
```suggestion
func TestDownloadOpts_PostRun(t *testing.T) {
```

names should be `Test[FuncName]`  for exported functions, `Test_funcName` for private functions, `TestStrucName_[F][f]uncName` for functions on a struct